packages/core/scripts/local/getMedianHistoricalPrice.js

```diff
@@ -3,14 +3,21 @@
  * It could be used for example to query a price before committing a vote for a DVM price request.
  * We provide default configurations for querying median prices on specific markets across some exchanges and markets.
  * This script should serve as a template for constructing other historical median queries.
- *  *
- * @dev How to run: $(npm bin)/truffle exec ./scripts/local/GetMedianHistoricalPrice.js --network mainnet --identifier <PRICE-FEED IDENTIFIER> --time <TIMESTAMP IN SECONDS>
+ *
+ * @notice This script will fail if the `--time` is not within a 4 day lookback window and one of the medianized pricefeeds is a cryptowatch price feed.
```

Comment:
@mrice32 FYI I've noticed that I can't get prices more than 4 days back. Is this expected even with the paid CW plan?

Reply:
Yeah, we probably just need to change how we query since it tries to request _all data_ between then and now (the `update()` + `get*()` pattern). My guess is they have more data, we're just requesting too much at once.

Reply:
Ah ok, will leave for another PR. What do you think the best way to space out the data is?